src/v/redpanda/admin/api-doc/debug_bundle.json

```diff
@@ -1,6 +1,6 @@
 "/v1/debug/bundle": {
   "post": {
-    "description": "Posting to this endpoint will trigger the start of an rpk debug bundle process.  If a process is already executing, this endpoint will return a 409 error.  If there is an issue with starting the process, this endpoint will return a 500 error.\n\nNote if job_id is not provided in BundleStartConfig, one will be automatically generated and returned in BundleStartResponse.",
+    "description": "Posting to this endpoint will trigger the start of an rpk debug bundle process.  If a process is already executing, this endpoint will return a 409 error.  If there is an issue with starting the process, this endpoint will return a 500 error.",
```

Comment:
chore: maybe combine these changes with the previous commit

Reply:
Done